Application/EdFi.Ods.Tests/EdFi.Ods.Common/Models/Domain/EntityTests.cs

```diff
@@ -609,25 +609,25 @@ protected override void Act()
                     {
                         new EntityPropertyDefinition(
                             "StudentUSI",
-                            new PropertyType(DbType.Int32, 0, 10, 0, false),
+                            new PropertyType(DbType.Int32, 0, 0, 10, 0, false),
```

Comment:
As I mentioned in an earlier comment, move the `minLength` argument to the end of the constructor signature and revert all changes such as this.

Reply:
I moved to last as minLength parameter  and updated it for all places at the end .